{"name": "test_update_post_with_invalid_id", "status": "skipped", "statusDetails": {"message": "XFAIL Fake API accepts string ID\n\nAssertionError: Expected 400 or 404 for invalid ID\nassert 500 in [400, 404]\n +  where 500 = <Response [500]>.status_code", "trace": "api = <api.posts_api.PostsAPI object at 0x000001B7E6B7E080>\n\n    @pytest.mark.negative_test\n    @pytest.mark.xfail(reason=\"Fake API accepts string ID\")\n    def test_update_post_with_invalid_id(api):\n        payload = negative_data[\"invalid_id_post\"]\n        res = api.update_post(\"abc\", payload)\n>       assert res.status_code in [400, 404], \"Expected 400 or 404 for invalid ID\"\nE       AssertionError: Expected 400 or 404 for invalid ID\nE       assert 500 in [400, 404]\nE        +  where 500 = <Response [500]>.status_code\n\ntests\\test_posts.py:83: AssertionError"}, "attachments": [{"name": "log", "source": "3dddd6b3-555a-451a-92f2-d1b2c7eaa010-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "0cc2fe39-8325-4fa6-a086-3425b93f37dd-attachment.txt", "type": "text/plain"}], "start": 1751386233167, "stop": 1751386233825, "uuid": "03193f64-38a6-4b19-b983-76cb603d698f", "historyId": "1c5e41e8af2790ca3aaa52307104bd68", "testCaseId": "1c5e41e8af2790ca3aaa52307104bd68", "fullName": "tests.test_posts#test_update_post_with_invalid_id", "labels": [{"name": "tag", "value": "@pytest.mark.xfail(reason='Fake API accepts string ID')"}, {"name": "tag", "value": "negative_test"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_posts"}, {"name": "host", "value": "DESKTOP-J0TB641"}, {"name": "thread", "value": "9736-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_posts"}]}